Message.php

```diff
@@ -235,6 +242,35 @@ protected function setBody($body, $contentType)
     }
 
     /**
+     * Returns a \Swift_Signers_DKIMSigner object
+     * @param type $privateKey
+     * @param type $domain
+     * @param type $selector
+     * @return \Swift_Signers_DKIMSigner
+     */
+    public function getDkimSigner($privateKey, $domain, $selector)
+    {
+        return new \Swift_Signers_DKIMSigner($privateKey, $domain, $selector);
+    }
```

Comment:
For testing. Also this allows the user to manually attach the signer to the message. For example if you have the key as a string instead of on the local filesystem.